File: java/logger/src/main/java/org/huba/logger/dto/PublishLogDto.java
package org.huba.logger.dto;

import lombok.Data;

import java.sql.Timestamp;
import java.time.LocalDateTime;
import java.util.List;
import java.util.UUID;

@Data
public class PublishLogDto {
    private UUID requestId;
    private String publishService;
    private RequestDto request;
    private ResponseDto response;
    private String otherInfo;
}

File: employee-gateway/EmployeeGateway/EmployeeGateway.Common/DTO/Logger/PublishLogDto.cs
namespace EmployeeGateway.Common.DTO.Logger;

public class PublishLogDto
{
    public string RequestId { get; set; }

    public string PublishService { get; set; }

    public RequestDto Request { get; set; }

    public ResponseDto Response { get; set; }

    public string otherInfo { get; set; }
}